declarations/LoaderContext.d.ts

```diff
@@ -0,0 +1,172 @@
+import type { AssetInfo, Configuration } from "../lib";
+import Compilation from "../lib/Compilation";
+import NormalModule, { InputFileSystem } from "../lib/NormalModule";
+import type { Mode } from "./WebpackOptions";
+
+export interface LoaderContext {
+	version: number;
+	getOptions(schema: any): any;
```

Comment:
You could steal the `Schema` type from `schema-utils`:

`Parameters<typeof import("schema-utils").validate>[0]`

Reply:
Will steal!